deploifai/api/__init__.py

```diff
@@ -599,3 +600,54 @@ def get_training_servers(self, workspace: str, where_training: dict = None):
         except KeyError:
             raise DeploifaiAPIError("Could not find Training Server. Please try again.")
 
+    def start_training_server(self, server_id: str):
+        mutation = """
+        mutation($where : String){
+            startTraining(where: {id: $where}){
```

Comment:
Should adhere closely to the `where` argument type which is `TrainingWhereUniqueInput` type, otherwise it's very confusing.

Reply:
Yes
Refactored